src/EntityGraphQL/Schema/MethodField.cs

```diff
@@ -84,6 +84,9 @@ ExecutionOptions executionOptions
 
         var graphQLArgumentsSet = new ArgumentsTracker();
 
+        IGraphQLValidator? validator = serviceProvider?.GetService<IGraphQLValidator>();
+        validator?.Errors.Clear(); // If the validator isn't registered as transient, we may have errors related to a different root field.
```

Comment:
Don't think we should clear errors. We should update docs to describe how to use and register `IGraphQLValidator`. Not clearing it allows someone to register it as not transient and maybe they want to fail everything if 1 fails 🤷